File: Python/Baju.py
# import class aksesoris dulu
from Aksesoris import Aksesoris

# class baju ini turunannya aksesoris
class Baju(Aksesoris):
    def __init__(self, id="", nama_produk="", harga_produk=0, stok_produk=0, jenis="", bahan="", warna="", untuk="", size="", merk=""):
        # panggil constructor dari aksesoris biar atribut umum keisi dulu
        super().__init__(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna)
        # atribut khusus buat baju
        self.__untuk = untuk
        self.__size = size
        self.__merk = merk

    # buat ngambil siapa yang pake bajunya (misal anjing, kucing, dll.)
    def get_untuk(self):
        return self.__untuk

    # buat ngubah siapa yang pake bajunya
    def set_untuk(self, untuk):
        self.__untuk = untuk

    # buat ngambil ukuran baju
    def get_size(self):
        return self.__size

    # buat ngubah ukuran baju
    def set_size(self, size):
        self.__size = size

    # buat ngambil merk baju
    def get_merk(self):
        return self.__merk

    # buat ngubah merk baju
    def set_merk(self, merk):
        self.__merk = merk

    # buat nampilin data baju
    def display(self):
        # panggil display dari aksesoris biar atribut umum tetep ditampilin
        super().display()
        # tambahin atribut spesifik baju
        print(f" | untuk: {self.__untuk} | size: {self.__size} | merk: {self.__merk}")

File: PHP/Baju.php
<?php
include_once "Aksesoris.php";

class Baju extends Aksesoris {
    private $untuk;
    private $size;
    private $merk;

    public function __construct($id = "", $nama_produk = "", $harga_produk = 0, $stok_produk = 0, $foto_produk = "", $jenis = "", $bahan = "", $warna = "", $untuk = "", $size = "", $merk = "") {
        parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna); // Panggil constructor Aksesoris
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }

    public function getUntuk() {
        return $this->untuk;
    }
    public function setUntuk($untuk) {
        $this->untuk = $untuk;
    }

    public function getSize() {
        return $this->size;
    }
    public function setSize($size) {
        $this->size = $size;
    }

    public function getMerk() {
        return $this->merk;
    }
    public function setMerk($merk) {
        $this->merk = $merk;
    }

    public function display() {
        parent::display();
        echo " | Untuk: {$this->untuk} | Size: {$this->size} | Merk: {$this->merk}\n";
    }
}
?>
